examples/qwen/conf/serve/serve_multiple_instance.yaml

```diff
@@ -0,0 +1,20 @@
+model_args:
+  vllm_model:
+    model: /models/DeepSeek-R1-Distill-Qwen-32B #Qwen2.5-7B-Instruct
+    tensor_parallel_size: 2
+    pipeline_parallel_size: 1
+    gpu_memory_utilization: 0.9
+    max_model_len: 32768
+    max_num_seqs: 256
+    enforce_eager: True
+    trust_remote_code: True
+    enable_chunked_prefill: True
+
+deploy:
+  models:
+    vllm_model:
+      num_replicas: 4
+      num_gpus: 2
```

Comment:
The variable `num_gpus` should be renamed to `num_gpus_per_replica`. Additionally, the `deploy` section needs to be redesigned.

Reply:
num_gpus is named the same as ray. we'd better keep it.